### PR TITLE
Link to the Akraino REC architecture and install guide

### PR DESCRIPTION
**Pull request by CsatariGergely**
Changing the link of the Akraino REC blueprint family to a link ot the Akraino REC architecture ans the intall guide.

**What type of PR is this?**
> Uncomment only one, leave it on its own line:
>
> bug
> cleanup
> design
documentation
> failing-test
> feature

**What does this PR give to us**:
Better links to Akraino REC

**Which issue(s) this PR fixes** *(in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
None

**Special notes for your reviewer**:
None

**Does this PR introduce a user-facing change?**:
None

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,8 +111,7 @@ That being said, tell us about your Kubernetes breaking ideas! We are open to ac
 Just because something doesn't fit into DANM, it does not mean it can't fit into your cloud! 
 ## Getting started
 ### Install an Akraino REC and get DANM for free!
-Just kidding as DANM is always free, but if you want to install a production grade, open-source Kubernetes-based bare metal CaaS infrastructure by default equipped with DANM **and** with a single click of a button nonetheless; just head over to Linux Foundation Akraino Radio Edge Cloud site:
-[Akraino REC Wiki](https://wiki.akraino.org/display/AK/Telco+Appliance+Blueprint+Family)
+Just kidding as DANM is always free, but if you want to install a production grade, open-source Kubernetes-based bare metal CaaS infrastructure by default equipped with DANM **and** with a single click of a button nonetheless; just head over to Linux Foundation Akraino Radio Edge Cloud (REC) wiki for the [Akraino REC Architecture](https://wiki.akraino.org/display/AK/REC+Architecture+Document) and the [Akraino REC Intallation Guide](https://wiki.akraino.org/display/AK/REC+Installation+Guide)
```
Comment (Levovar) [inline README.md]: *In**s**tallation ;)

**Comment by pcarver** [comment]
Do you want to add the caveat that the Akraino REC installer is a hardware opinionated option? I'm obviously an advocate of REC, but it is very picky about hardware. Wouldn't want to give people a negative DANM experience just because they don't have the right hardware to install REC.
Remember the old quote "Unix is user-friendly — it's just choosy about who its friends are."

**Comment by CsatariGergely** [comment]
It depends on how match we would like to explain the details of Akraino REC here. I would rather add the details to the Akraino REC Install guide and add a maximum half sentence note to the DANM page. @Levovar what do you think?

**Comment by Levovar** [comment]
My intentions were only basically "linking" our reference integration. I think everyone can decide for themselves if it is easier to install a REC, or they would rather build up their own cluster.
Even if they end up building their own cluster, the open source code can be still useful to see how others have integrated DANM into production

**Pull request merged**